content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,55 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
+
+Scan folders can be added as registry settings. In most cases you'll use the asset folders of Gems or your project, which will be automatically added as scan folders. However, if you need to add a new scan folder for tracking source assets, you can add it under the registry setting path `Amazon/AssetProcessor/Settings/` with a pattern of:
+```json
+"ScanFolder _scan folder name here_": {
+  "watch": "_path to watch folder_",
+  "display": "_display name for scan folder_",
+  "recursive": 0 or 1, // 1 will tell the Asset Processor to look in sub folders, recursively for additional source assets. 0 it will not.
+  "order": any number // The order this scan folder is handled compared to other scan folders.
+}
+```
+Scan folder order is used when two assets have the exact same relative path to the scan folder root. In these situations, the source asset in the scan folder with the higher priority will be used over the other one.
+
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
-
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
-
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
-
-* Add scan directories for Asset Processor.
-
-* Set the number of process jobs.
-* Specify *metafile fingerprinting*, that is, which files trigger related files to be rebuilt.
-
-The `Registry/AssetProcessorPlatformConfig.setreg` is well commented and has many syntax examples for the available configuration options. To add project-specific overrides, you can add an `AssetProcessorGamePlatformConfig.setreg` file to your projects’s `Registry` directory. This file is read after the engine `AssetProcessorGamePlatformConfig.setreg` configuration file and can have additional project-specific settings for the ignore list, platforms, and file types.
\ No newline at end of file
+You can use the exclusion system to block files matching patterns from asset processing. These exist under the registry path Amazon/AssetProcessor/Settings/ with a pattern that looks like:
+```json
+"Exclude _exclusion title_": {
+  "pattern": "regular expression pattern, anything that matches this pattern will be ignored by the Asset Processor"
+}
+```
```

Comment:
````suggestion
The exclusion system is used to block files matching patterns from asset processing. These exist under the registry path `Amazon/AssetProcessor/Settings/` with a pattern that looks like:

```json
"Exclude <Exclusion title>": {
  "pattern": "<regex of files to exclude>"
}
```
````

Reply:
I've cleaned this up to match your suggestion